todo-app/src/App.js

```diff
@@ -1,13 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
+import {AppTask} from './AppTask';
```

Comment:
When creating React components in a single file, it's common practice to set that component as the default export, so you can just do:

    import AppTask from './AppTask'

The only time where it's appropriate to use named exports like this is if we were exporting _multiple_ components from a single file.

Check out [this Stackoverflow post](https://stackoverflow.com/questions/33611812/javascript-es6-export-const-vs-export-default), it's a pretty good explanation.

In your case, all you have to do is add `default` to your exported components (I'll also leave a comment there), and change the import syntax here.